### DISCUSSION TITLE
Inaccurate Net Worth on Dashboard

### DISCUSSION THREAD

**Discussion by drewzeee** [firefly-iii/firefly-iii#9187] [Firefly III]
As title describes, the Net Worth is showing an inaccurate amount. Not sure how this is being calculated. In my example, NW is showing as $15,635 while the total sum of my accounts is only $11,144.51. I checked to make sure that all accounts are being included in net worth. What should I look at next?
<img width="235" alt="Screenshot 2024-09-01 at 10 33 25 AM" src="https://github.com/user-attachments/assets/65f104b1-7d65-4cca-924d-05d8adee4bc6">
<img width="128" alt="Screenshot 2024-09-01 at 10 34 00 AM" src="https://github.com/user-attachments/assets/f43b84a0-d80e-437e-a2e2-bc44c3ecf494">

**Comment by JC5** [comment]
Hey, great question! That particular sum is the balance difference, not the account balance. Those are two different numbers.

**Reply by drewzeee** [reply]
Thanks for the reply! I apologize but I still don't understand. Perhaps it's because I wasn't very clear in my first post. 
I currently have 4 accounts. A checking, a savings, And two credit cards (setup as asset accounts). The current sum total of all four accounts is the $11k figure, so that is my actual net worth by normal definitions. 

Are you saying that the net worth figure on the dashboard(the $15k figure) is the difference from a separate amount?

**Reply by JC5** [reply]
No, I see it now, I'm the confused one. Sorry!

Not sure what the cause is. Could be cache? The math behind it is excruciatingly simple. I'll check it out.

**Reply by drewzeee** [reply]
I thought it might be cache too. Cleared it, but the discrepancy remains. Purged deleted records too. Let me know if I can provide any other info!

**Reply by Administratoor** [reply]
Hi, the networth also respects the date-range set on the top-right of the page!
To see what's the networth of each account is, navigate to http://YOUR_FIREFLY_DOMAIN/accounts/asset page. This page also uses the datepicker. So you can timetravel in past and in the future. If you have some regular income records in the future they may be calculated for your networth.

You can also look at the standard finance report. It has a networth graph in it. There you may find at which time your networth was or will be that high.
Hope you can find an answer, (or help to find the bug)

**Reply by CoffeeViking** [reply]
@JC5 Is there a way to just set the "Net Worth" value to a sum of my accounts instead of being tied to a report or however it's being drummed up now? I may be getting a similar bug to OP here too because the NW value doesn't line up with the report it links to either. 

Either way, for my purposes it'd be easier/better for me if I could track that as a basic summation vs the income/expense calcs that it's doing now. I couldn't find anything about doing so, figured I'd ask about it. :)

Thanks for the project though, it's pretty nice overall.

**Reply by JC5** [reply]
No, there isn't. You need Excel for that.
